app/models/video.py

```diff
@@ -0,0 +1,19 @@
+from flask import current_app
+from app import db
+
+class Video(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime, nullable=True)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer, default=0)
```

Comment:
Like with `Customer` we can compute `available_inventory` with a relationship:

```
rentals = db.relationship('Rental', back_populates='video', lazy=True)
```

Like with `Customer` the `available_inventory` is the total inventory minus length of the `rentals` that haven't been checked in:

```
"available_inventory": self.total_inventory - len(rental for rental in self.rentals if not rental.outstanding)
```